packages/rancher-istio/rancher-tracing/charts/templates/deployment.yaml

```diff
@@ -74,10 +74,14 @@ spec:
         runAsUser: 1000
       serviceAccountName: {{ include "tracing.fullname" . }}
       {{- end }}
-      {{- with .Values.tolerations }}
-      tolerations:
-        {{- toYaml . | nindent 8 }}
-      {{- end }}
+      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
+{{- if .Values.nodeSelector }}
+{{ toYaml .Values.nodeSelector | indent 8 }}
+{{- end }}
+      tolerations: {{ include "linux-node-tolerations" . | nindent 8 }}
+{{- if .Values.tolerations }}
+{{ toYaml .Values.tolerations | indent 8 }}
+{{- end }}
```

Comment:
nodeSelectors were missing, tolerations were broken.

note: affinity here only does arch, not OS